package.json

```diff
@@ -52,7 +52,7 @@
     "apollo-client": "^0.4.12"
   },
   "devDependencies": {
-    "apollo-client": "0.4.13",
+    "apollo-client": "0.4.16",
```

Comment:
I fixed the version here to test it with the faulty version of **apollo-client**, but I think we should change it to the latest version. These changes should back compatible, because `reduxRootKey` has precedence over `reduxRootSelector`.